frontend/src/Editor/Inspector/Components/FilePicker.jsx

```diff
@@ -27,7 +27,7 @@ export const FilePicker = ({ componentMeta, darkMode, ...restProps }) => {
 
     options.map((option) => renderOptions.push(renderCustomElement(option)));
 
-    const conditionalOptions = [{ name: 'parseFileType', condition: parseContent }];
+    const conditionalOptions = [{ name: 'parseFileType', condition: parseContent === '{{true}}' }];
```

Comment:
If `parseContent` is programmatically configured (eg: `variables.someVariable > 33.3`), this condition will be erroneous won't it?

Resolve it using resolveReferences and then check if it is `true`.